src/main/java/com/rocketden/main/service/UserService.java

```diff
@@ -53,10 +53,10 @@ public UserDto createUser(CreateUserRequest request) {
     }
 
     public UserDto deleteUser(DeleteUserRequest request) {
-        User user = repository.findUserByUserId(request.getUserId());
+        User user = UserMapper.toEntity(request.getUserToDelete());
 
-        // If requested user does not exist in database, throw an exception. 
-        if (user == null) {
+        // If requested user does not exist in database, throw an exception.
+        if (repository.findUserByUserId(user.getUserId()) == null) {
```

Comment:
I made this change after switching the fields in the DeleteUserRequest from strings to UserDto. Looking back I could have just changed the User assignment line instead of also the if statement.